src/features/presence/PresenceAvatar.js

```diff
@@ -26,6 +42,7 @@ const PresenceAvatar = ({
         onFocus={onFocus}
         onMouseEnter={onMouseEnter}
         onMouseLeave={onMouseLeave}
+        role="figure"
```

Comment:
Is this correct for an interactive element? I think we've been removing the use of `figure` in most places, as well. It also looks like the line below has a condition of `isDropDownAvatar`.